server/controllers/reservationController.js

```diff
@@ -116,4 +116,28 @@ const createReservation = async (reservation) => {
   }
 }
 
-module.exports = { getNearestGarages, getAllReservations, testAllReservations, createReservation };
\ No newline at end of file
+/*
+  query contains { reservation_id, status } both strings
+  reservation id is from the path query and status is from the params
+  This is consolidated in the route handler and passed here to the controller
+  status must be one of the following ['checked-in', 'picked-up', 'cancelled'];
+*/
+const updateReservation = async (query) => {
+  console.log(query)
```

Comment:
You can leave the console logs in this line and line 134 for now until the bug has been resolved/we meet with client to demo